### DISCUSSION TITLE
How do I become a contributor ?

### DISCUSSION THREAD

**Discussion by tariqhamid** [BitMaker-hub/NerdMiner_v2#389] [General]
I have a pending pull request that adds support for ESP32-8048S070C and ESP32-2432S028C

How do I add push PR to NerdMiner_v2 ?

**Comment by tariqhamid** [comment]
Please respond ?

**Reply by BitMaker-hub** [reply]
Hello, Past month I was fully concentrated on NerdAxe. I'm sorry.
I'll start merging all PRs the next incoming weeks, and I'll make a new release with all the new supported boards
